README.md

```diff
@@ -1,12 +1,10 @@
 # react-a11y-announcer
 
-Welcome to React Announcer from Think Company.
```

Comment:
I think this line should stay, since it modestly and demurely mentions who we are, setting context for the "we" below.